blocks/blockstore/bloom_cache.go

```diff
@@ -17,9 +20,29 @@ func bloomCached(bs Blockstore, ctx context.Context, bloomSize, hashCount int) (
 		return nil, err
 	}
 	bc := &bloomcache{blockstore: bs, bloom: bl}
+	bc.hits = metrics.NewCtx(ctx, "bloom.hits_total",
+		"Number of cache hits in bloom cache").Counter()
+	bc.total = metrics.NewCtx(ctx, "bloom_total",
+		"Total number of requests to bloom cache").Counter()
+
+	fill := metrics.NewCtx(ctx, "bloom_fill_ratio",
+		"Ratio of bloom filter fullnes, (updated once a minute)").Gauge()
+
 	bc.Invalidate()
 	go bc.Rebuild(ctx)
-
+	go func() {
```

Comment:
Yeah, had the same idea, I will add method to go-metrics-interface to check if there was some impl injected.